### DISCUSSION TITLE
install had a problem /root/sosetup.log

### DISCUSSION THREAD

**Discussion by ffcolman** [Security-Onion-Solutions/securityonion#6356] [Unsupported Versions]
I have been trying various options but the issue persists. I will keep trying to analyze the log in detail.....

I7 4 cores/12Mbram/200GbHd/Eval version/Vbox/VMware.After screen error, run so-status showing error in MySql, started and fixed.
Of course the installation is from a ISO file,  

thanks in advance for any clue..
f

**Comment by dougburks** [comment]
From https://github.com/Security-Onion-Solutions/securityonion/discussions/1720:

Please include the following details where you can:

- Security Onion version as seen in the lower left corner of SOC and in /etc/soversion. For example: 2.3.90
- Is this a cloud deployment or on-prem?
- Did you install from our Security Onion ISO image or did you perform a network installation?
- If network installation, did you install on CentOS 7 or Ubuntu?
- How many nodes do you have? How are each of those nodes configured? (ex. manager with 2 search nodes and 3 forward nodes)
- Are you experiencing issues monitoring network traffic? If so, are you sniffing from a tap or span port and what is the traffic volume?
- Does so-status show all services running?
- Do you get any failures when you run sudo salt-call state.highstate?
- Does the SOC Grid page show any failures?
- Explain your issue. For example: Installation fails when I select this series of options...
- Provide applicable logs. For example: if you are having problems during soup, provide /root/soup.log
